lib/Listener/FlagRepliedMessageListener.php

```diff
@@ -97,6 +97,9 @@ public function handle(Event $event): void {
 						'exception' => $e,
 					]);
 				}
+
+				$message->setFlagAnswered(true);
```

Comment:
:warning: you are outside the loop. `$message` is undefined if you have 0 messages or you access the *last* one in the `$messages` array.

Reply:
Line 103 is the loop end bracket tho?

Reply:
![Bildschirmfoto vom 2022-04-08 17-55-03](https://user-images.githubusercontent.com/1374172/162478577-098594c7-35bc-4a0d-a1d4-efc8976a1249.png)

stupid wonky github indentation :see_no_evil: